app/src/main/java/net/gsantner/markor/activity/DocumentEditFragment.java

```diff
@@ -669,12 +669,13 @@ public String getFragmentTag() {
     // Save the file
     // Only supports java.io.File. TODO: Android Content
     public boolean saveDocument(boolean forceSaveEmpty) {
+
+        _appSettings.setLastEditPosition(_document.getFile(), _hlEditor.getSelectionStart());
+
         // Document is written iff content has changed
         // _isTextChanged implies _document != null && _hlEditor != null && _hlEditor.getText() != null
         if (_isTextChanged && isAdded()) {
 
-            _appSettings.setLastEditPosition(_document.getFile(), _hlEditor.getSelectionStart());
```

Comment:
@gsantner I think this may address the issue you were seeing with position not being saved.